Formula/sane-backends.rb

```diff
@@ -45,17 +50,70 @@ def install
 end
 
 __END__
```

Comment:
Can remove the `__END__` and the entire diff now you're using the `patch do` format.